src/Cedar/Proto_OpenVPN.h

```diff
@@ -184,10 +184,6 @@
 #define	OPENVPN_MODE_L2							1		// TAP (Ethernet)
 #define	OPENVPN_MODE_L3							2		// TUN (IP)
 
-// Data
-#define OPENVPN_DATA_OPTIONS	0
-#define OPENVPN_DATA_PEERINFO	1
-

```

Comment:
These lines were supposed to be removed in 9970d6f657aa4abc08bd1719083394af791c0e25, sorry for the mistake.